app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+end
```

Comment:
- [ ] As the previous reviewer suggested, please implement a method that updates the likes counter for a post 🙏